README.md

```diff
@@ -420,13 +420,72 @@ leftwm-theme apply NAME_OF_THEME_YOU_LIKE
 
 ## Without [LeftWM-Theme](https://github.com/leftwm/leftwm-theme)
 
+To set up your own custom theme, you will need to create a directory containing the theme files and then symlink it to `$XDG_CONFIG_HOME/leftwm/themes/current`.
+
+A theme directory contains at least 1 file:
+
+- `up`: a script which is loaded with the theme.
+
+Many theme directories also contain:
+
+- `down`: a script which is called when the theme is unloaded.
+- `theme.ron`: which contains additional configuration options specific to the theme.
+- `polybar.ini`: a configuration file for the `polybar` application. *Need to have polybar installed!*
+- `picom.conf`: a configuration file for the `picom` compositor. *Need to have picom installed!*
+
+Here is a prototypical example of an `up` file.
```

Comment:
Shouldn't we rather reference the `themes` folder of the repo here?

Reply:
Yes perhaps that is better. I just thought this information would have useful to me when setting everything up.

Reply:
I am just talking about the example files, because there are lots of them in the `themes` folder. I like the explanation of how it works.